libnavui-maneuver/src/main/java/com/mapbox/navigation/ui/maneuver/model/RoadShield.kt

```diff
@@ -1,13 +1,17 @@
 package com.mapbox.navigation.ui.maneuver.model
 
+import com.mapbox.api.directions.v5.models.MapboxShield
+
 /**
  * Data structure  that holds information about road shield.
  * @property shieldUrl String
  * @property shieldIcon ByteArray
+ * @property mapboxShield MapboxShield
  */
 data class RoadShield(
```

Comment:
Why do we use `data class`? With its `copy` method there's no way to update `RoadShield` without breaking SemVer.

Reply:
Since we're deprecating the `RoadShieldCallback` callback anyway, I think we should deprecate `RoadShield` and `RoadShieldError` as well and introduce a non-data classes.

Being able to receive Mapbox Design shield requires a change anyway (passing user and style IDs in a request), so we could require transition to a new callback and type at the same time.